src/main/java/org/threadly/litesockets/utils/SSLUtils.java

```diff
@@ -61,6 +82,70 @@ public static void enableSNI() {
     System.setProperty ("jsse.enableSNIExtension", "true");
   }
   
+  public static String fileToString(File file) throws IOException {
+    RandomAccessFile raf = new RandomAccessFile(file, "r");
```

Comment:
because I hate input/output streams, just a personal thing, much prefer old school c type fopen type systematics for files.  imo normal file system files are not streams they are blocks and should be treated that way.

Do you think I should do any size checking?  the files should be relativity small. so ensure they are < 1mb or something?